packages/openapi-to-graphql/README.md

```diff
@@ -137,34 +137,50 @@ Notice that the slashes in the path `/favoriteBooks/{name}` must be escaped with
 The `createGraphQlSchema` function takes an optional `options` object as a second argument:
 
 ```javascript
-createGraphQLSchema(oas, [options])
+createGraphQLSchema(oas, options)
 ```
 
 The options object can contain the following properties:
 
 - `strict` (type: `boolean`, default: `false`): OpenAPI-to-GraphQL generally tries to produce a working GraphQL schema for a given OAS if the strict mode is disabled. If OpenAPI-to-GraphQL cannot fully translate a given OAS (e.g., because data schema definitions are incomplete or there are name collusions that cannot be resolved), `createGraphQLSchema` will per default degrade gracefully and produce a partial GraphQL schema. OpenAPI-to-GraphQL will log warnings (given logging is enabled). If the `strict` mode is enabled, however, `createGraphQLSchema` will throw an error if it cannot create a GraphQL schema matching the given OAS perfectly.
 
-- `headers` (type: `object`, default: `{}`): Headers to be sent in every request to the API described by the given OAS. Parameters defined in the OpenAPI Specification to set these headers will be ignored by OpenAPI-to-GraphQL.
-
-- `qs` (type: `object`, default: `{}`): Query parameters to be sent in every request to the API described by the given OAS. Parameters defined in the OpenAPI Specification to set these query parameters will be ignored by OpenAPI-to-GraphQL.
+***
 
-- `viewer` (type: `boolean`, default: `true`): The viewer object types (i.e. `QueryViewer` and `MutationViewer`) are artificial constructs that allow users to pass authentication credentials to OpenAPI-to-GraphQL. Unfortunately, they are bulky and do not provide an accurate representation of the API. Depending on the API, it may be possible to send all credentials through the header option, so this option allows to disable OpenAPI-to-GraphQL-generated viewer object types.
+Schema options:
 
-- `tokenJSONpath` (type: `string`, default: `undefined`): Used to pass the [JSONPath](http://goessner.net/articles/JsonPath/) of the OAuth token in the GraphQL context. To see more details, click [here](./README.md#authorization).
+- `operationIdFieldNames` (type: `boolean`, default: `false`): By default, query field names are based on the return type type name and mutation field names are based on the [`operationId`](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#operation-object), which may be generated if it does not exist. This option forces OpenAPI-to-GraphQL to only create field names based on the operationId.
```

Comment:
type type

Reply:
I think this is intentional... as in "the name of the type (aka type name), specifically the type that is returned (aka return type)". What do you think?

Reply:
Then `by the return type's type name`?